client/layout/guided-tours/site-title-tour.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	Next,
+	Quit,
+	Continue,
+	Link,
+	ButtonRow,
+} from 'layout/guided-tours/config-elements';
+import {
+	hasSelectedSiteDefaultSiteTitle,
+	isUserOlderThan,
+	isEnabled,
+	canUserEditSettingsOfSelectedSite,
+	isAbTestInVariant,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+import { isDesktop } from 'lib/viewport';
+
+const TWO_DAYS_IN_MILLISECONDS = 2 * 1000 * 3600 * 24;
+
+export const SiteTitleTour = makeTour(
+	<Tour
+		name="siteTitle"
+		version="20161010"
+		path="/stats"
+		when={ and(
+						isEnabled( 'guided-tours/site-title' ),
+						isDesktop,
+						hasSelectedSiteDefaultSiteTitle,
+						canUserEditSettingsOfSelectedSite,
+						isUserOlderThan( TWO_DAYS_IN_MILLISECONDS ),
+						isAbTestInVariant( 'siteTitleTour', 'enabled' )
+					)
```

Comment:
The indentation is off (made more obvious in Github due to 8-char tabs). Indenting just one level after `and(` is probably enough. Also, `)` and `}` should belong on the same line, leaving `>` on its own, and at the same level as the opening `<Tour`